src/solver/mha/mha_ck_fa_v2_solver_forward.cpp

```diff
@@ -240,7 +240,8 @@ MhaCKFlashAttentionV2Forward::GetSolution([[maybe_unused]] const ExecutionContex
 
             fmha_runtime_args.p_drop = probability;
             fmha_runtime_args.drop_seed_offset =
-                std::make_pair(dataFwd.dropoutSeedData, dataFwd.dropoutOffsetData);
+                std::make_pair(reinterpret_cast<uint64_t>(dataFwd.dropoutSeedData),
```

Comment:
@CAHEK7 had to this to silence the compiler.

Reply:
I'd like to double check - are you going to convert gpu pointer to integer? Is the code using `fmha_runtime_args.drop_seed_offset` aware that it's actually gpu pointer?
For me that `reinterpret_cast` doesn't look like a correct solution.

Reply:
yes! its a GPU pointer but I think for now since we have disabled the dropout https://github.com/ROCm/MIOpen/blob/develop/src/solver/mha/mha_ck_fa_v2_solver_forward.cpp#L234

Reply:
It would be better to make it explicitly 0 and put the same comment as 
https://github.com/ROCm/MIOpen/blob/0c24160d7368f1194cf24cea713af82ead4275c2/src/solver/mha/mha_ck_fa_v2_solver_forward.cpp#L237-L239